.github/workflows/mt.yaml

```diff
@@ -52,12 +56,13 @@ jobs:
             composer-
 
       - name: Install dependencies
-        run: |
-          composer update --no-interaction --prefer-dist --no-progress
+        # Run the locked dependencies: we are not interested in testing different
+        # variations here so having a stable set is better.
+        run: composer install --no-interaction --prefer-dist --no-progress
 
       - name: Collect coverage report
         run: |
-          php vendor/phpunit/phpunit/phpunit --stop-on-failure \
+          SYMFONY_DEPRECATIONS_HELPER=disabled php vendor/phpunit/phpunit/phpunit --stop-on-failure \
```

Comment:
why is it needed?

Reply:
Because depending on the deprecations it can fail this build. It's not necessarily bad, but we already check for the deprecations in `ci.yaml`. So having it here is redundant, and disabling it guaranteeing we get to the interesting part of that build which is running infection with coverage generated and with the badge API key

Reply:
Lemme know if that's a satisfying answer for you or you prefer to keep it